CoreFoundation/Locale.subproj/CFLocale.c

```diff
@@ -1170,25 +1426,98 @@ static bool __CFLocaleCopyCollatorID(CFLocaleRef locale, bool user, CFTypeRef *c
     *cf = canonLocaleCFStr;
     return canonLocaleCFStr ? true : false;
 }
-    
+
+#if DEPLOYMENT_TARGET_MACOSX || DEPLOYMENT_TARGET_EMBEDDED
+static CONST_STRING_DECL(_metricUnitsKey, "AppleMetricUnits");
+static CONST_STRING_DECL(_measurementUnitsKey, "AppleMeasurementUnits");
+static CONST_STRING_DECL(_measurementUnitsCentimeters, "Centimeters");
+static CONST_STRING_DECL(_measurementUnitsInches, "Inches");
+static CONST_STRING_DECL(_temperatureUnitKey, "AppleTemperatureUnit");
+
+static bool __CFLocaleGetMeasurementSystemForPreferences(CFTypeRef metricPref, CFTypeRef measurementPref, UMeasurementSystem *outMeasurementSystem) {
+    if (metricPref || measurementPref) {
+        if (metricPref == kCFBooleanTrue && measurementPref && CFEqual(measurementPref, _measurementUnitsInches)) {
+            *outMeasurementSystem = UMS_UK;
```

Comment:
This broke the build on Ubuntu 14.04 because UMS_UK is not defined. Can we revert this for now?

Reply:
I’d prefer not to and to instead ifdef out the usage

Reply:
It looks like @slavapestov reverted the commit in

https://github.com/apple/swift-corelibs-foundation/commit/60b7bd22a8f699ae00d5b25bcd2c371f7d1bd4b9

So this will need to be re-reverted again at some point.